sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/ShareFileStorageResourceOptions.cs

```diff
@@ -90,7 +172,6 @@ public class ShareFileStorageResourceOptions
         ///
         /// Applies to download transfers.
         /// </summary>
-        public DownloadTransferValidationOptions DownloadTransferValidationOptions
-        { get; set; }
+        public DownloadTransferValidationOptions DownloadTransferValidationOptions { get; set; }
```

Comment:
Unrelated to this PR: Did we get rid of these in Blob?

Reply:
We did! Good catch, I will probably make another quick PR to remove here too.

Reply:
Addressing it in this PR https://github.com/Azure/azure-sdk-for-net/pull/43841/files